api/src/main/resources/vwb-user-manager.yaml

```diff
@@ -2164,14 +2164,12 @@ components:
         createdDate:
           description: Timestamp of creation
           type: string
-          format: date-time
```

Comment:
This caused an issue while deserializing the response because the returned timestamp doesn't match the expected OffsetDateTime format. Removing this format makes it just a String, which we can format manually if we need to. Currently we don't need these fields entirely